static/extensions/pooiod/Box2D.js

```diff
@@ -685,29 +676,46 @@ while keeping general compatability. (made with box2D js es6) */
             },
           },
 
-          
           {
-            hideFromPalette: !physdebugmode && !wipblocks,
-            blockType: Scratch.BlockType.LABEL, // --------------------- Work in progress blocks ----
-            text: "Upcoming blocks (can brake projects)"
+            hideFromPalette: !legacymode,
+            blockType: Scratch.BlockType.LABEL,  // ---- Legacy blocks (old and not recomended for use) ----
+            text: `Legacy blocks`
           },
           {
-            opcode: 'ignore',
```

Comment:
same as above - dont delete/change block opcodes

Reply:
the ignore block was a leftover debug block, and should never be used in an actual project (text above it stated that the usage of it could result in projects braking)